src/devices/Ssd1351/Ssd1351Commands.cs

```diff
@@ -0,0 +1,427 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.Runtime.InteropServices;
+
+namespace Iot.Device.Ssd1351
+{
+
+    public enum Ssd1351Command : byte
```

Comment:
Should this enum be public? You seem to have a wrapper function for each one of this which also include param validation some times so I'm not sure if it makes sense to expose a different way of doing it as well by calling SendCommand and passing one of these.

Reply:
I hadn't wrapped all the commands but now I have. and so the enum is now internal in the latest update.